device/bgp/bgpadvanced.yaml

```diff
@@ -45,3 +45,20 @@ components:
             If not configured, MD5 authentication will not be enabled.
           type: string
           x-field-uid: 5
+        passive_mode:
+          description: |-
+            If enabled then the BGP Finite State Machine will passively wait 
+            for the remote BGP peer to establish the BGP TCP connection. 
+            If the passive mode is true, BGP peer will wait for the remote BGP peer 
+            to issue an Open request before an Open message is sent.
+          type: boolean
+          default: false
+          x-field-uid: 6
+        local_listen_port:
+          description: |-
+            Set listen transport port number for the local BGP peer.
```

Comment:
Assuming this new attribute is used for both connecting and accepting BGP connections , the description may need to be updated to something like : 
Set the TCP port number to be used for both accepting BGP connections from remote peer as well as to initiate connections to remote peer.  [ Else for two separate attributes we need to break the description accordingly as well ]. 
Note sure if we need to specifically mention that if an implementation has limitations e.g. all sessions initiated from same test port needing to have same port number , set_config  should return error specifying the same.
Maybe if not in the model, this can be specified in PR notes before sending to wider team.

Reply:
Also needs a linked Issue as discussed and code snippet after changes if any can be put inside code format in description.